net/net_test.go

```diff
@@ -21,6 +21,185 @@ import (
 	"testing"
 )
 
+func TestParseCIDRs(t *testing.T) {
+	testCases := []struct {
+		cidrs         []string
+		errString     string
+		errorExpected bool
+	}{
+		{
+			cidrs:         []string{},
+			errString:     "should not return an error for an empty slice",
+			errorExpected: false,
+		},
+		{
+			cidrs:         []string{"10.0.0.0/8", "not-a-valid-cidr", "2000::/10"},
+			errString:     "should return error for bad cidr",
+			errorExpected: true,
+		},
+		{
+			cidrs:         []string{"10.0.0.0/8", "2000::/10"},
+			errString:     "should not return error for good  cidrs",
+			errorExpected: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		cidrs, err := ParseCIDRsStrings(tc.cidrs)
+		if tc.errorExpected {
+			if err == nil {
+				t.Errorf("%v", tc.errString)
+			}
```

Comment:
`continue` and then you don't have to use else and indent the rest of the block (fix in other tests as well)